docs/source/overview.rst

```diff
@@ -0,0 +1,305 @@
+
+- Clusters
+- Jobs
+- Services
+
+These abstractions allow you to run all use cases in the AI lifecycle:
+Batch processing, development, (pre)training, finetuning, hparam sweeps, batch inference, and online serving.
+
+In SkyPilot, every workload benefits from:
+
+.. dropdown:: Unified execution on any cloud, region, and cluster
+
+    Regardless of how many clouds, regions, and clusters you have, you can use a unified interface
+    to submit, run, and manage workloads on them.
+
+    Your focus on the workload, and SkyPilot alleviates the burden of
+    dealing with cloud infra details and differences.
+
+.. dropdown:: Cost and capacity optimization
+
+    When launching a workload, SkyPilot will automatically choose the cheapest and most available infra choice in your search space.
+
+.. dropdown:: Auto-failover across infra choices
+
+    When launching a workload, you can give SkyPilot a search space of infra
+    choices --- as unrestricted or as specific as you like.
+
+    If an infra choice has no capacity,
+    SkyPilot automatically falls back to the next best choice in your infra search space.
+
+.. dropdown:: No cloud lock-in
+
+    Should you add infra choices (e.g., a new cloud, region, or cluster) in the future, your existing workloads can easily run on them.
+    No complex migration or workflow changes.
+    See the underlying :ref:`Sky Computing <sky-computing>` vision.
+
+.. _concept-dev-clusters:
+
+Clusters
+------------
+
+A *cluster* is SkyPilot's core resource unit: one or more VMs or Kubernetes pods in the same location.
+
+You can use ``sky launch`` to launch a cluster:
+
+.. tab-set::
+
+    .. tab-item:: CLI
+        :sync: cli
+
+        .. code-block:: console
+
+            $ sky launch
+            $ sky launch --gpus L4:8
+            $ sky launch --num-nodes 10 --cpus 2+ --memory 2+ --down
+            $ sky launch cluster.yaml
+            $ sky launch --help  # See all flags.
+
+    .. tab-item:: Python
+        :sync: python
+
+        .. code-block:: python
+
+            import sky
+            task = sky.Task().set_resources(sky.Resources(accelerators='L4:8'))
+            sky.launch(task, cluster_name='my-cluster')
+
+You can do the following with a cluster:
+
+- SSH into any node
+- Connect VSCode/IDE to it
+- Submit and queue many jobs on it
+- Have it automatically shut down or stop to save costs
+- Easily launch and use many virtual, ephemeral clusters
+
+
+You can optionally bring your custom Docker or VM image when launching, or use SkyPilot's sane defaults, which configure the correct CUDA versions for different GPUs.
+
+See :ref:`quickstart` and :ref:`dev-cluster` to get started.
+
+.. tip::
+
+    Think of clusters as *virtual* in nature. They can be launched on *physical*
+    clusters you bring to SkyPilot, such as :ref:`Kubernetes clusters
+    <concept-kubernetes-clusters>` or :ref:`existing machines
+    <concept-existing-machines>`.
```

Comment:
I would rather include cloud VM in the sentence to avoid confusion:

```suggestion
    A SkyPilot cluster is a *virtual* collection of cloud VMs (instances) or pods launched on *physical*
    clusters you bring to SkyPilot, such as :ref:`Kubernetes clusters
    <concept-kubernetes-clusters>` or :ref:`existing machines
    <concept-existing-machines>`.
```